src/Microsoft.AspNetCore.SignalR.Client/HubConnectionBuilder.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.ComponentModel;
+using System.Net.Http;
+using Microsoft.AspNetCore.SignalR.Internal.Protocol;
+using Microsoft.AspNetCore.Sockets;
+using Microsoft.AspNetCore.Sockets.Client;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+
+namespace Microsoft.AspNetCore.SignalR.Client
+{
+    public class HubConnectionBuilder
```

Comment:
One downside with this approach is that it isn't extensible. The other builders we made exists so that they can be extended by extension methods outside of this assembly. The Core HubConnectionBuilder would be in SignalR.Client.Core and there would be extension methods that add the http* on top.